WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -501,6 +512,25 @@ fileprivate func < <T: Comparable>(lhs: T?, rhs: T?) -> Bool {
         saveForLaterButton.isSelected = postIsSavedForLater
     }
 
+    fileprivate func configureReblogActionButton() {
+        reblogActionButton.isHidden = !shouldShowReblogActionButton
+    }
+
+    fileprivate var shouldShowReblogActionButton: Bool {
+        guard FeatureFlag.postReblogging.enabled else {
+            return false
+        }
+        // No post content -> nothing to reblog
+        guard contentProvider != nil else {
+            return false
+        }
+        // User not logged in -> User cannot reblog
+        guard loggedInActionVisibility.isEnabled else {
+            return false
```

Comment:
Can we have just one `guard` statement for all these `false` conditions?

Reply:
👍